object-storage-tck/src/main/groovy/io/micronaut/objectstorage/ObjectStorageOperationsSpecification.groovy

```diff
@@ -77,7 +77,12 @@ abstract class ObjectStorageOperationsSpecification extends Specification {
         when: 'updating the file'
         if (emulatorSupportsUpdate()) {
             path.toFile().text = NEW_TEXT
-            uploadRequest = UploadRequest.fromPath(path)
+            if (key.contains(File.separator)) {
```

Comment:
I wonder about the `File.separator` here. In upload requests are you supposed to use OS specific separators? Wouldn't it always be `/`?

Reply:
Although you are correct, note this is mostly test logic